internal/pkg/template/diff/diff.go

```diff
@@ -201,7 +203,10 @@ func parseSequence(fromNode, toNode *yaml.Node, overrider overrider) ([]diffNode
 	}
 	cachedDiff := make(map[string]cachedEntry)
 	lcsIndices := longestCommonSubsequence(fromSeq, toSeq, func(idxFrom, idxTo int) bool {
-		diff, err := parse(&(fromSeq[idxFrom]), &(toSeq[idxTo]), "", overrider)
+		if diff, ok := cachedDiff[cacheKey(idxFrom, idxTo)]; ok {
+			return diff.err == nil && diff.node == nil
```

Comment:
nit: can you add a comment here on what scenario this represents? 🥺

Reply:
done!